tests/ptvsd/server/test_start_stop.py

```diff
@@ -58,8 +62,7 @@ def code_to_debug():
 
 
 @pytest.mark.skipif(
-    sys.version_info < (3, 0) and platform.system() == "Windows",
-    reason="On Windows + Python 2, unable to send key strokes to test.",
+    sys.version_info < (3, 0), reason="https://github.com/microsoft/ptvsd/issues/1819"
```

Comment:
Fixing #1819 will not fix the Windows + Py 2.7 issue. It should skip for this separately?

Reply:
I changed the press any key code to detect when the terminal is redirected, and use "Press Enter" rather than "Press any key" code path for that. So it should work even on Windows now - the reason why it didn't despite the fix is because of #1819.

https://github.com/microsoft/ptvsd/blob/a2d58ee7c925fe90313b9d28cd771090357a2f9d/src/ptvsd/launcher/debuggee.py#L147-L152